package.json

```diff
@@ -13,14 +13,17 @@
         "postinstall": "npm run boot",
         "test": "lerna run test",
         "publish:check": "lerna changed",
-        "publish": "lerna publish"
+        "publish:release": "lerna publish",
```

Comment:
Do we want to keep this one around, if we are always planning on going through the publish script?

Reply:
The publish script calls ``publish:release``.

Reply:
https://github.com/google/blockly-samples/pull/247/files#diff-b9e12334e9eafd8341a6107dd98510c9R72